Windows build errors

E.g. [here](https://github.com/ForNeVeR/tdlib.native/actions/runs/3610858075/jobs/6084913458):
```
cli.obj : error LNK2019: unresolved external symbol rl_line_buffer referenced in function "void __cdecl td::deactivate_readline(void)" (?deactivate_readline@td@@YAXXZ) [D:\a\tdlib.native\tdlib.native\td\build\tg_cli.vcxproj]
cli.obj : error LNK2019: unresolved external symbol rl_point referenced in function "public: virtual __cdecl td::LambdaGuard<class <lambda_8722b28637a2d18e5dc1f1ead801aff1> >::~LambdaGuard<class <lambda_8722b28637a2d18e5dc1f1ead801aff1> >(void)" (??1?$LambdaGuard@V<lambda_8722b28637a2d18e5dc1f1ead801aff1>@@@td@@UEAA@XZ) [D:\a\tdlib.native\tdlib.native\td\build\tg_cli.vcxproj]
cli.obj : error LNK2019: unresolved external symbol rl_end referenced in function "void __cdecl td::deactivate_readline(void)" (?deactivate_readline@td@@YAXXZ) [D:\a\tdlib.native\tdlib.native\td\build\tg_cli.vcxproj]
cli.obj : error LNK2019: unresolved external symbol rl_getc_function referenced in function "private: virtual void __cdecl td::CliClient::loop(void)" (?loop@CliClient@td@@EEAAXXZ) [D:\a\tdlib.native\tdlib.native\td\build\tg_cli.vcxproj]
cli.obj : error LNK2019: unresolved external symbol rl_attempted_completion_function referenced in function "private: virtual void __cdecl td::CliClient::loop(void)" (?loop@CliClient@td@@EEAAXXZ) [D:\a\tdlib.native\tdlib.native\td\build\tg_cli.vcxproj]
D:\a\tdlib.native\tdlib.native\td\build\Release\tg_cli.exe : fatal error LNK1120: 5 unresolved externals [D:\a\tdlib.native\tdlib.native\td\build\tg_cli.vcxproj]
```

There weren't any updates from my side, so perhaps something was changed after agent update.

--- a/windows/build.ps1
+++ b/windows/build.ps1
@@ -21,6 +21,7 @@ try {
     )
     $cmakeArguments = @(
         "-DCMAKE_TOOLCHAIN_FILE=$VcpkgToolchain"
+        '-DCMAKE_DISABLE_FIND_PACKAGE_Readline=TRUE' # workaround for #76
         '..'
     )
     $cmakeBuildArguments = @(

